docs/DynamicPartitionedAvroFileSet-batchsink.md

```diff
@@ -0,0 +1,56 @@
+# Dynamic PartitionedFileSet Avro Batch Sink
+
+
+Description
+-----------
+Sink for a ``PartitionedFileSet`` that writes data in Avro format
+and leverages one or more record field values for creating partitions.
+All data for the run will be written to a partition based on the
+specified fields and their value.
+
+
+Use Case
+--------
+This sink is used whenever you want to write to a ``PartitionedFileSet`` in Avro format
+using a value from the record as a partition. For example, you might want to load historical
+data from a database and partition the dataset on the original creation date of the data.
+
+
+Properties
+----------
+**name:** Name of the ``PartitionedFileSet`` to which records are written.
```

Comment:
Could you convert these properties into this table format? here and in the docs page.